data/human/intro missions.txt

```diff
@@ -11,6 +12,455 @@
+	choice
+		`	"Of course."`
+		`	"Sorry, but I was planning on heading out in a different direction from here."`
+			decline
+	branch paradise
+		has "start: paradise"
+	`	Once again you walk back to your ship with James. Once you get there, as he's stowing his luggage, you pull up the information your computer has on <planet>. It's one of the "Paradise Worlds," terraformed to have a perfect climate, accessible only to the very wealthy. You wonder whether James is going to be able to fit in there.`
+		accept
+	label paradise
+	`	Once again you walk back to your ship with James. You've heard of <planet> before; one of the Paradise Worlds terraformed from a lifeless ice planet to the largest and most luxurious retirement home in the galaxy. As you look at his utilitarian outfit and modest luggage, something seems off about the idea of James living in such a place.`
+		accept
+
+
+
+conversation "freighter 1 end"
+	`You drop off the shipment of <commodity> on <planet>, and collect your payment of <payment>. "Hauling freight," says James. "Not the most glamorous job, or the most interesting, but you're always guaranteed steady income. And once you know the galaxy a bit better, instead of taking missions you can find your own trade routes to follow. The only trouble is, build up a fleet with lots of cargo space and not enough defenses and the pirates begin to take notice."`
+	choice
+		`	"Did you have a lot of trouble with pirates when you were a captain?"`
+		`	"But what if I want to earn money more quickly than I can by trading?"`
+			goto quick
+	`	"Some. That's why you can't just go traipsing around the galaxy in a fleet of unarmed star barges; every pirate within a week's travel will hear you're coming through and come after you. In the long term, having a few heavy freighters with good defenses is more cost-effective than losing a star barge or two to the pirates on every journey."`
+		goto port
+	label quick
+	`	"There's no quick road to riches, kid. The more something pays, the riskier it is. If you need to make lots of money in a hurry, bounty hunting is the way to go, but nine out of ten pilots who give bounty hunting a try end up dead. And, no one's going to give you those sorts of missions until you've proved yourself in combat, anyway."`
+	label port
+	`	He picks up his suitcase. "If you want to run another mission with me, look for me in the spaceport. I'll see if I can find you something with a higher profit margin this time."`
+	``
+	`	(As before, click the "Spaceport" button to continue this mission.)`
+
+
+
+conversation "freighter 2 end"
+	`By now, your entire ship smells like brine and raw fish, but as best you can tell the fish is still fresh enough to eat. The dock workers on <planet> offload it without comment, and give you your payment of <payment>.`
+		to display
```

Comment:
Why do these lines differ depending on your start? To be honest I don't really see why they would be different.